### ISSUE TITLE
Why  Cmd + Q is not available with role = "quit"

### ISSUE THREAD

**Issue by rjyo** [blackboardsh/electrobun#28]
I have some DB logic in my Next.js. I can have it export to standalone mode and run my service with just `bun server.js`, which will work in browser at http://localhost:3000.

The question is how can I package it using electrobun?

**Comment by rjyo** [comment]
Thanks for the great lib! It turns out to be fairly easy:

Here's the entrypoint:
```typescript
import { BrowserWindow, ApplicationMenu } from "electrobun/bun";

ApplicationMenu.setApplicationMenu([
  {
    submenu: [{ label: "Quit", role: "quit" }],
  }
]);

const path = "../standalone/server.js";
import(path);

const win = new BrowserWindow({
  title: "reb",
  url: "http://localhost:3000",
});

win.on("close", () => {
  console.log("close");
  process.exit(0);
});
```

And here's the electrobun.config:
```
{
  "app": {
    "name": "My App",
    "identifier": "dev.my.app",
    "version": "0.0.1"
  },
  "build": {
    "bun": {
      "entrypoint": "scripts/index.ts"
    },
    "copy": {
      ".next/standalone": "standalone"
    }
  }
}
```

**Comment by rjyo** [comment]
The only problem I have now is Cmd + Q is not available. I tried other menu role like copy, that works with shortcut perfectly.

**Comment by YoavCodes** [comment]
Great timing, I'm working on something in [co(lab)](https://colab.sh) where I needed a custom global shortcut even when a `<electrobun-webview>` has focus. just published electrobun 0.0.17 that should fix this.

```
 {
    label: "co(lab)",
    submenu: [{ role: "quit", accelerator: "q" }],
  },
```

Note: it seems on macos that a small `q` results in `cmd + q` and a capital `Q` results in `cmd + shift + q`. I'll need to write some docs but in the meantime hopefully it unblocks

**Comment by rjyo** [comment]
Thank you @YoavCodes It worked like a charm!
